### DISCUSSION TITLE
What is the future of the Gatsby Open Source project?

### DISCUSSION THREAD

**Discussion by visualjeff** [gatsbyjs/gatsby#38668] [Umbrella Discussions]
- Original thought leaders have moved on.
- Individual contributors have been laid off.
- Today's contributors are primary github maintenance bots.
- Issues starting to pile up.

What is the state of Gatsby?

Is anyone else concerned?   Or are developers transitioning their projects to other frameworks?

**Comment by zhaohuanyuu** [comment]
I am also worried that the community is not as active as before. I am ready to migrate to astro
It's so sad...

**Reply by visualjeff** [reply]
Appreciate you taking the time to reply.

On the positive side, at least we have options.

**Comment by Isaac-Tait** [comment]
This discussion question and recently reading [this on Reddit](https://www.reddit.com/r/gatsbyjs/comments/17fiw6a/are_you_worried_about_the_future_of_gatsby/) has me even wondering if I should even bother with [this issue I am having ](https://github.com/gatsbyjs/gatsby/discussions/38688)at all and just jump ship and switch my project over to NextJS...
